source/cloud-native-platform/guides/postgresql-singleserver-to-flexibleserver-migration.erb

```diff
@@ -0,0 +1,33 @@
+---
+title: Postgresql single server to flexible server migration using Azure DMS 
+last_reviewed_on: 2023-04-24
+review_in: 6 months
+weight: 10
+hide_in_navigation: true
+---
+
+# <%= current_page.data.title %>
+
+Azure Database Migration Service is an universal tool used to migrate databases to Azure platform. For migrating larger databases(> 50GB) it is advised to use this tool as it has minimal downtime. 
+
+### Limitations
+
+- Can't handle database of sizes > 1 TB
+- Can't handle write-intensive workload
+- You can have only one active migration to your flexible server.
+- You can select a max of eight databases in one migration attempt. If you've more than eight databases, you must wait for the first migration to be complete before initiating another migration for the rest of the databases. Support for migration of more than eight databases in a single migration will be introduced later.
+- The source and target server must be in the same Azure region. Cross region migrations are not supported.
+- The tool takes care of the migration of data and schema. It doesn't migrate managed service features such as server parameters, connection security details, firewall rules, users, roles and permissions. In the later part of the document, we point you to docs that can help you perform the migration of users, roles and firewall rules from single server to flexible server.
+- The migration tool shows the number of tables copied from source to target server. You need to validate the data in target server post migration.
+- The tool only migrates user databases and not system databases like template_0, template_1, azure_sys and azure_maintenance.
+
+### Prerequisite 
+
+- Create an `Azure Resource Group`
+- Update "out of scope" list by adding above Resource Group in azure-policy repo for respective subscription.
+        [eg.]( https://github.com/hmcts/azure-policy/pull/200)
```

Comment:
I don't think we would create this every time wouldn't we just have a shared one and stop it when not using it?